shared/message/platform/RawSensors.proto

```diff
@@ -32,21 +32,44 @@ message RawSensors {
     /// Set of error flags/statuses.
     /// bitmask values.
 
-    /// @note these are kind of out of date or are triggered elsewhere in Dynamixel v2
-    ///       ANGLE_LIMIT, CORRUPT_DATA and TIMEOUT won't get triggered
-    ///       RANGE, CHECKSUM and INSTRUCTION are processed per status-return packet so
-    ///       will never make it to a RawSensors message.
+    /// @note RawSensors::Error is kept for protocol v1 compatibility only.
+    /// Values are prefixed with an underscore to resolve naming conflicts.
```

Comment:
I think leading underscores are reserved or something? What is it conflicting with?

Reply:
oh whoops, i changed the name later so the conflict wasn't an issue anymore, forgot to change the underscores back

Reply:
the RawSensors::Error enum has names shared with RawSensors::PacketError and RawSensors::HardwareError, Can change the underscore to something else

Reply:
leading underscore has been replaced by a trailing underscore to avoid using reserved identifiers